pages/blog.vue

```diff
@@ -46,10 +45,8 @@ export default {
   async fetch () {
```

Comment:
I think we should implement the reusable function for the fetch medium article, then, we can use it in TopArticles file and blog file

Reply:
@minapham17 Ok, I have created a mixin to reuse this fetch method in both components.

Reply:
@minapham17 I agree with you. It's done, now any part of the whole article card is a link to medium article.